app/service/service_broadcast_settings_schema.py

```diff
@@ -0,0 +1,12 @@
+service_broadcast_settings_schema = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Set a services broadcast settings",
+    "type": "object",
+    "title": "Set a services broadcast settings",
+    "properties": {
+        "broadcast_channel": {"enum": ["test", "severe"]},
+        "service_mode": {"enum": ["training", "live"]},
+        "provider_restriction": {"enum": [None, "three", "o2", "vodafone", "ee"]}
```

Comment:
Should it be `null` instead of `None`? I haven't got the admin part of this to try it out, so wasn't sure if it needs to be something like `"type": ["string", "null"], "enum": [ null, "three", "o2", "vodafone",  "ee"]`

Reply:
Good question and I'm not 100% sure. Given that my local dev works fine when the admin app posts to it, I think it's OK.

I think what is happening is that when I send `None`, when it's parsed into JSON, it becomes `null`. And then when it's parsed back to python it becomes `None` and that appears to be validated correctly against the list of enum options.

If I change it to `"provider_restriction": {"type": ["null"]}` and then run `test_set_as_broadcast_service_updates_mobile_provider_restriction` it passes for the `None` case. 

Therefore I think if I changed it to `"type": ["string", "null"], "enum": [ None, "three", "o2", "vodafone", "ee"]` then it would be equivalent to what I already have written.

Note, `"type": ["string", "null"], "enum": [ null, "three", "o2", "vodafone", "ee"]` would be invalid because `null` is not defined and this is python code being evaluated.

Reply:
Oh, the other thing I could have done is to remove `provider_restriction` from the list of required keys but the downside of this is that then it makes it a bit opaque that if you didn't include this key, it might remove any current setting in the database for the provider restriction for this service. I think therefore it's good to make the caller of the API be very explicit about what they want to change by including the `provider_restriction` key